embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/SchemaType.kt

```diff
@@ -195,6 +197,20 @@ internal sealed class SchemaType(
         ).toNonNullMap()
     }
 
+    internal class NetworkRequest(networkRequest: EmbraceNetworkRequest) : SchemaType(EmbType.Performance.Network) {
+        override val attrs = mapOf(
+            "url.full" to networkRequest.url,
```

Comment:
we'll move to the semantic conventions if they can fix it in time - otherwise, we can move these into `OpenTelemetryAttributeKeys` later